src/components/utils/popover/popover-desktop.ts

```diff
@@ -309,50 +346,96 @@ export class PopoverDesktop extends PopoverAbstract {
    *
    * @param item - item to display nested popover by
    */
-  private showNestedPopoverForItem(item: PopoverItemDefault): void {
+  private showNestedPopoverForItem(item: WithChildren<PopoverItemDefault> | WithChildren<PopoverItemHtml>): void {
     this.nestedPopover = new PopoverDesktop({
+      searchable: item.isChildrenSearchable,
       items: item.children,
       nestingLevel: this.nestingLevel + 1,
     });
 
+    this.emit(PopoverEvent.OpenNestedPopover, {
+      /** Inputs potentially can be auto focused */
+      hasAutoFocusableElements: this.nestedPopover.renderParams.containsInputs,
+    });
+
+    /**
+     * Close nested popover when item with 'activateOnClose' property set was clicked
+     * parent popover should also be closed
+     */
+    this.nestedPopover.on(PopoverEvent.CloseOnActivate, this.hide);
+
     const nestedPopoverEl = this.nestedPopover.getElement();
 
     this.nodes.popover.appendChild(nestedPopoverEl);
-    const itemEl =  item.getElement();
-    const itemOffsetTop = (itemEl ? itemEl.offsetTop : 0) - this.scrollTop;
-    const topOffset = this.offsetTop + itemOffsetTop;
 
-    nestedPopoverEl.style.setProperty('--trigger-item-top', topOffset + 'px');
-    nestedPopoverEl.style.setProperty('--nesting-level', this.nestedPopover.nestingLevel.toString());
+    this.setTriggerItemPositionProperty(nestedPopoverEl, item);
+
+    nestedPopoverEl.style.setProperty(CSSVariables.NestingLevel, this.nestedPopover.nestingLevel.toString());
```

Comment:
add doc please, why CSS should know about nesting level

Reply:
done